editor/plugins/script_editor_plugin.cpp

```diff
@@ -1593,6 +1593,12 @@ void ScriptEditor::ensure_select_current() {
 
 			if (!grab_focus_block && is_visible_in_tree())
 				se->ensure_focus();
+
+			if (Object::cast_to<ScriptTextEditor>(se)) { //sadly this is necessary, because code_editor is not a property of ScriptEditorBase
+				Object::cast_to<ScriptTextEditor>(se)->get_code_editor()->set_find_replace_bar(shared_find_replace_bar);
```

Comment:
This should be `Object::cast_to<ScriptTextEditor>(se)->set_find_replace_bar(shared_find_replace_bar);` rather then exposing `CodeEditor`